README.md

```diff
@@ -244,7 +244,7 @@ Validators:
 - [X] length 
 - [X] type 
 - [X] numericality (greater than, equal to, is integer, etc)
-- [ ] format - regex
+- [X] format - regex
```

Comment:
Also, there is the `Null Values` section which must inform how `format` validator deals with `null` and `undefined`

Reply:
I didn't understand what you need to do in this section of code.